lib/workers/repository/init/config.js

```diff
@@ -53,39 +55,58 @@ async function mergeRenovateConfig(config) {
     if (!renovateConfig.length) {
       renovateConfig = '{}';
     }
-    let allowDuplicateKeys = true;
-    let jsonValidationError = jsonValidator.validate(
-      renovateConfig,
-      allowDuplicateKeys
-    );
-    if (jsonValidationError) {
-      const error = new Error('config-validation');
-      error.configFile = configFile;
-      error.validationError = 'Invalid JSON (parsing failed)';
-      error.validationMessage = jsonValidationError;
-      throw error;
-    }
-    allowDuplicateKeys = false;
-    jsonValidationError = jsonValidator.validate(
-      renovateConfig,
-      allowDuplicateKeys
-    );
-    if (jsonValidationError) {
-      const error = new Error('config-validation');
-      error.configFile = configFile;
-      error.validationError = 'Duplicate keys in JSON';
-      error.validationMessage = JSON.stringify(jsonValidationError);
-      throw error;
-    }
-    try {
-      renovateJson = JSON.parse(renovateConfig);
-    } catch (err) /* istanbul ignore next */ {
-      logger.debug({ renovateConfig }, 'Error parsing renovate config');
-      const error = new Error('config-validation');
-      error.configFile = configFile;
-      error.validationError = 'Invalid JSON (parsing failed)';
-      error.validationMessage = 'JSON.parse error: ' + err.message;
-      throw error;
+
+    const fileType = path.extname(configFile);
+
+    if (fileType !== '.json5') {
```

Comment:
As a best practice, it's better to not use a negation in an if/else statement. i.e. write:

```
if (foo === bar) {
  // do something if true
} else {
  // do something if not true
}
```

And more confusing to write:
```
if (foo !== bar) {
  // do something if not true
} else {
  // do something if true
}
```